liquibase/changeLogs/capture/data/aqToNwisMethodCode/aqToNwisMethodCode.tsv

```diff
@@ -21,3 +21,4 @@ GW level, pressure gage	Pressure-gage measurement.	G
 GW level, pressure transducer	Transducer.	F
 GW level, steel tape	Steel-tape measurement.	S
 GW level,unvented press transdcr	Transducer.	F
+None	null	Z
```

Comment:
Does lowercase 'null' work (probably does). The doc I have seen just have uppercase 'NULL'

Reply:
Good catch, yes it works.